velox/expression/tests/ArrayProxyTest.cpp

```diff
@@ -278,5 +278,43 @@ TEST_F(ArrayProxyTest, testTimeStamp) {
   assertEqualVectors(result, makeNullableArrayVector(expected));
 }
 
+TEST_F(ArrayProxyTest, testString) {
+  auto result =
+      prepareResult(std::make_shared<ArrayType>(ArrayType(VARCHAR())));
```

Comment:
Can we add a test for varbinary too? Just to ensure the types are set correctly.

Reply:
yep